include/aspect/material_model/grain_size.h

```diff
@@ -36,6 +37,7 @@ namespace aspect
   namespace MaterialModel
   {
     using namespace dealii;
+    using VectorType = Vector<double>;
```

Comment:
I don't think you use this anywhere outside the one function where you call ARKode. Can you move that there (i.e., into the function)?